app/main/view.ts

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+import { BrowserView } from 'electron';
+
+import ConfigUtil = require('../renderer/js/utils/config-util');
+const shouldSilentWebview = ConfigUtil.getConfigItem('silent');
+
+export interface PropsType {
+	index: number;
+	url: string;
+	name: string;
+	nodeIntegration: boolean;
+	preload: boolean;
+}
+
+export class View extends BrowserView {
+	props: PropsType;
+	index: number;
+	url: string;
+	zoomFactor: number;
+	loading: true;
+	badgeCount: number;
+	customCSS: boolean;
+
+	constructor(props: PropsType) {
```

Comment:
Just a syntactic sugar you can do here: `constructor(public props: ViewProps)` and then remove `this.props = props;` below.

Reply:
Done.